### DISCUSSION TITLE
蓝灯老是自动切服务器，然后就无法链接

### DISCUSSION THREAD

**Discussion by hacker873** [getlantern/lantern-forum-cn#989] [General]
蓝灯自动切服务器的问题都快一年了，还能不能改好了，老是切服务器，然后就断网无法链接，真后悔又续了两年，你们要还是解决不了退款吧。

**Comment by MaxMKMax** [comment]
@hacker873 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by hacker873** [reply]
现在上传不了，网页蓝灯页面灰色，中间圆圈一直在转，啥也点不了，能上网就算不错了。

**Comment by MaxMKMax** [comment]
@hacker873 之前有弄好了, 但最近又有这个问题了. 我再帮你们追问一下看能否再次修复好. 估计翻墙变难了, 所以才会再次出现这个问题. 非常抱歉啊, 希望您耐心等待. 谢谢您啊!

**Reply by hacker873** [reply]
前段时间还好点，这几天蓝灯连接都是问题，服务器不断的跳转，连接上没有数据，网页显示的蓝灯自动刷新，最主要是把我其它页面也给刷新了，还是不断的重复刷新，搞得不用代理就能看的网站也没法正常访问

**Comment by MaxMKMax** [comment]
@hacker873 您好. 请问您有截图吗? 给我几个截图一下. 谢谢.
